start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
+[servers:vars]
+pbench_repo_url_prefix = https://copr-be.cloud.fedoraproject.org/results/ndokos
+
+# where to get the key
+pbench_key_url = http://example.com/some/path/to/agent/{{ pbench_configuration_environment }}/ssh
+
+# where to get the config file
+pbench_config_url = http://example.com/some/path/to/agent/{{ pbench_configuration_environment }}/config
+                           </pre>
                          </p>
-			 <p class="c_snip">
-                           configfile=/path/to/config/file/pbench-agent.cfg
-                           <br></br>
-                           keyfile=/path/to/pbench/private/key/id_rsa
-                           <b></b>
-			   pbench-agent-config-activate $configfile
-                           <b>pbench-agent-config-ssh-key $configfile $keyfile</b>
+                         <p class="info_text"  style="text-align:justify">
+                           The above spec covers three hosts. Please make sure that the hosts are
+                           correct. A useful way to make sure that you are touching the correct
+                           set of hosts is the following command:
                          </p>
+                         <p class="info_text"  style="text-align:justify">
+                           <p class="c_snip">
+                           cd /path/to/some/directory/pbench/agent/ansible
+                           ansible-playbook -i ~/.config/Inventory/myhosts.inv --list-hosts pbench_agent_install.yml
+                           </p>
+                         </p>
+                         <p class="info_text"  style="text-align:justify">
+                           If you want to install on the local machine, the inventory file should include <span class="c_snip_inline">localhost</span> in
+                           the <span class="c_snip_inline">[servers]</span> section. Note that you need to be able to ssh to the localhost as root as well,
+                           so you might need to do <span class="c_snip_inline">ssh-copy-id root@localhost</span>.
+                         </p>
+                       </div>
+                     </div>
+                   </div>
+                   <!-- Subsection: Run the playbook -->
+		   <div class="accordionCard">
+		     <h4 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Run the playbook</span>
+		       </button>
+		     </h4>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+		       <div class="pf-c-accordion__expanded-content-body">
 			 <p class="info_text"  style="text-align:justify">
-                           You should be ready to roll.
+                           You can install <span class="c_snip_inline">pbench-agent</span> by using the following command:
```

Comment:
Perhaps "the pbench-agent RPM"?

Reply:
Actually, here we install not just the RPM, but the config file and the key file as well, so I rephrased accordingly.